src/Service.Tests/SqlTests/RestApiTests/Insert/DwSqlInsertApiTests.cs

```diff
@@ -1,274 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-using System;
-using System.Collections.Generic;
-using System.Net;
-using System.Threading.Tasks;
-using Azure.DataApiBuilder.Config.ObjectModel;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-namespace Azure.DataApiBuilder.Service.Tests.SqlTests.RestApiTests.Insert
-{
-    /// <summary>
-    /// Test REST Apis validating expected results are obtained.
```

Comment:
Why are we removing Insert tests for DW? In the config, you can keep the allowed rest methods to be `get`, `post` instead of just `get`, right?

Reply:
Rohan's reply: https://github.com/Azure/data-api-builder/pull/2048#discussion_r1520500939

Reply:
Please let me know if that is ok and we can move with completing this pr.